README.md

```diff
@@ -29,6 +30,38 @@ and we'll take a look.
 > build this site. Often, you can make changes using the GitHub UI.
 > If you want/need to build, read on.
 
+## Before you submit a PR
+
+We love it when the community gets involved in improving
+our docs! But here are a few notes to keep in mind before
+you submit a PR:
+
+* If you have an issue with the API docs on
+  [api.flutter.dev](https://api.flutter.dev),
+  please file those issues on the
+  [flutter/flutter](https://github.com/flutter/flutter/issues)
+  repo, not on this (flutter/website) repo.
+  The API docs are embedded in Flutter's source code,
+  so the engineering team handles those.
```

Comment:
This doesn't seem to belong in this section. Maybe add a **Before you submit an issue** section?